### ISSUE TITLE
Dojo build failing for 'hosted'

### ISSUE THREAD

**Issue by peller** [maqetta/maqetta#3568]
dojo_1_8 sources are not getting built for the hosted zip

@jhpedemonte

**Comment by jhpedemonte** [comment]
Here's what I see in the log files:  There are 2 "dojo.build" sections -- the first for non-hosted and the second for the hosted build.  The first section shows content, meaning that it is actually working.  The 2nd section (hosted) does not show any content.

**Comment by jhpedemonte** [comment]
Ignore my previous comment; the "dojo.build" sections related to the _maqetta.core.client_ plugin.

The Dojo build for the Dojo library bundle is handled by the **widgetPluginDojoBuild.xml** ant script.  The first pass (non-hosted build) builds Dojo and caches it in another dir.  The 2nd pass doesn't perform the build, instead copying it from the cache dir.

But here's the build log corresponding to when the files are copied into the cache dir:

```
restore.dojoLibs:
   [delete] Deleting directory /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/dijit
   [delete] Deleting directory /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/dojo
   [delete] Deleting directory /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/dojox
    [mkdir] Created dir: /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/dojo

cache.davinciPlugin.library:

cache.davinciPlugin.metadata:
     [copy] Copying 411 files to /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/dojo_cache/dojo_1_8/metadata
   [delete] Deleting directory /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/release
   [delete] Deleting directory /var/lib/jenkins/jobs/maqetta-orion/workspace/I201211271201/plugins/davinci.dojo_1_8/davinci.dojo_1_8_1.0.0.201211271221/WebContent/maq-metadata-dojo
```

Notice that `cache.davinciPlugin.library` is empty -- I would expect to see output similar to `cache.davinciPlugin.metadata`.  I can't see anything out of the ordinary in the ant script, unless the `<fileset>` element for `cache.davinciPlugin.library` is incorrect.

**Comment by jhpedemonte** [comment]
I have no idea why, but this seems to have been fixed by commit e9cdd65ade6b920e053a3a891b38c9d240f305ae!
